### PR TITLE
fix(documentation): remove reference to dev mode

### PR DESCRIPTION
**Pull request by modularTaco**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/PhotoboothProject/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [x] Documentation update
- [ ] Bug fix
- [ ] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)

According to https://github.com/PhotoboothProject/photobooth/issues/11 the dev mode has been removed. Instead the same result of setting photobooth to dev mode can be achieved by setting the loglevel to 1.  
So i removed the reference to the dev mode in the FAQ, as this setting is not available anymore.  
Further more the log level can be set in the basic administration mode already, no need to enable the expert view

### CODE REVIEW

File: faq/faq.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ Photobooth will watch GPIOs for a PIN_DOWN event - so the hardware button needs
 **Important: For WLAN connected screens you must make sure to set the IP address of the Photobooth web server in the admin settings - section "General"**. The loopback IP (127.0.0.1) does not work, it has to be the exact IP address of the Photobooth web server, to which the remote display connects to.
 
 Having trouble?
-- Switch Photobooth to DEV mode. (admin screen -> expert view -> general section)
+- Set Photobooth Log Level to 1. (admin screen -> general section)
```
Comment (andi34) [inline faq/faq.md]: to 1 (or above).

**Review by andi34 — CHANGES_REQUESTED** [review]
Thanks for the finding! Could you please adjust mentioned note?

**Comment by modularTaco** [comment]
> Thanks for the finding! Could you please adjust mentioned note?

Sure 👍

**Comment by modularTaco** [comment]
Hey @andi34 , [here](https://github.com/PhotoboothProject/photobooth/blob/dev/resources/lang/en.json#L313) is also a reference to dev mode. Can the language file be modified here in git directly, or has this to be done in the translation tool?

**Comment by andi34** [comment]
English can be modified here :)

**Comment by andi34** [comment]
Great! Thanks a lot!

Could you rebase your changes on top of latest dev branch please to avoid the commits `Merge branch 'PhotoboothProject:dev' into patch-2` ?

**Comment by modularTaco** [comment]
> Great! Thanks a lot!
> 
> Could you rebase your changes on top of latest dev branch please to avoid the commits `Merge branch 'PhotoboothProject:dev' into patch-2` ?

Won't they be gone already if you merge the pr and squash the commits?  
But sure, i rebased the branch on top of the latest commit in the dev branch

**Comment by andi34** [comment]
Squash merge was disabled, but enabled it and used it now =)
Thanks a lot again!

**Pull request merged**